corehq/apps/style/templates/style/bootstrap3/base.html

```diff
@@ -473,6 +482,32 @@
         {% endcompress %}
         {% endif %}
 
+
+        {% if request.use_maps or request.use_maps_async %}
+        {% compress js %}
+        <script type="text/javascript" src="{% static 'jquery-color/jquery.color.js' %}"></script>
+        <script type="text/javascript" src="{% static 'leaflet/dist/leaflet.js' %}"></script>
+        {% endcompress %}
+        {% endif %}
+
+        {% if reqeust.maps_prefer_canvas %}
+        <script>L_PREFER_CANVAS = true;</script>
+        {% endif %}
+
+        {% if request.use_maps %}
+        {% compress js %}
+        <script src="{% static 'reports/javascripts/maps.js' %}"></script>
+        <script src="{% static 'reports/javascripts/maps_utils.js' %}"></script>
+        {% endcompress %}
+        {% endif %}
+
+        {% if request.use_maps_async %}
+        {% compress js %}
+        <script src="{% static 'reports/javascripts/async_maps.js' %}"></script>
+        <script src="{% static 'reports/javascripts/maps_utils.js' %}"></script>
+        {% endcompress %}
+        {% endif %}
```

Comment:
Moving these here seems to have impacted the timing for when they get loaded: http://manage.dimagi.com/default.asp?222977

It looks like the partial page is evaluated before these are loaded. Locally the JS files are loaded first so I'm not able to reproduce.